test/fw/ptl/lib/pbs_api_to_cli.py

```diff
@@ -278,6 +278,8 @@
     ATTR_NODE_License: 'license',
     ATTR_NODE_LicenseInfo: 'license_info',
     ATTR_NODE_TopologyInfo: 'topology_info',
+    ATTR_NODE_last_used_time: 'last_used_time',
+    ATTR_NODE_last_state_change_time: 'last_state_change_time',
```

Comment:
I thought this was pulled out into a different PR

Reply:
Rebasing with master messed up my branch somehow. So I did a reset of this file with master. If you look at overall changes you should not see these changes.

Reply:
Cool thanks.  I took a look and it doesn't show up now.